homeassistant/components/decora_wifi/__init__.py

```diff
@@ -1 +1,108 @@
 """The decora_wifi component."""
+
+import logging
+
+from decora_wifi import DecoraWiFiSession
+from decora_wifi.models.iot_switch import IotSwitch
+from decora_wifi.models.permission import Permission
+from decora_wifi.models.person import Person
+from decora_wifi.models.residence import Residence
+from decora_wifi.models.residential_account import ResidentialAccount
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    EVENT_HOMEASSISTANT_STOP,
+    Platform,
+)
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+
+from .config_flow import BASE_SCHEMA
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = [Platform.LIGHT]
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
Don't need this one anymore iirc

Reply:
https://github.com/home-assistant/core/pull/104573/commits/bdc54d73df3c1a04ec5d118cf6a0bbeb204001de